tests/system-tests/rdscore/internal/rdscorecommon/egress-ip.go

```diff
@@ -626,23 +635,31 @@ func gracefulNodeReboot(nodeName string) error {
 		})
 
 	if err != nil {
-		glog.V(rdscoreparams.RDSCoreLogLevel).Infof("The node %s hasn't reached Ready state", nodeName)
+		glog.V(100).Infof("The node %s hasn't reached Ready state", nodeName)
 
 		return fmt.Errorf("node %s hasn't reached Ready state", nodeName)
 	}
 
-	glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Uncordoning node %q", nodeName)
+	glog.V(100).Infof("Uncordoning node %q", nodeName)
 
 	err = nodeObj.Uncordon()
 
 	if err != nil {
-		glog.V(rdscoreparams.RDSCoreLogLevel).Infof("Failed to uncordon %q due to %v", nodeName, err)
+		glog.V(100).Infof("Failed to uncordon %q due to %v", nodeName, err)
 
 		return fmt.Errorf("failed to uncordon %q due to %w", nodeName, err)
 	}
 
 	time.Sleep(15 * time.Second)
 
+	err = restoreActivePDBValues(pdbList)
```

Comment:
this part won't be executed in case there's any preceding error (draining/uncording/etc) so more appropriate place for this will be in _BeforeAll_ to capture initial state of PDBs and _Before/AfterEach_ to restore them, wdyt?

Reply:
I think it's a great idea, but I doubt how I can implement it. The problem is pdbList, all info for this List was collected during set... state

Reply:
actually I can separate 'collect pdb info' into stand alone function, wait a min, I'll check...

Reply:
no, it will not help. I still do not see how I can pass to the AfterEach state PDB list. Do you have any idea?

Reply:
@yprokule do you have any idea how to implement it?

Reply:
Sent U some draft directly. Also noticed that some _PDBs_ have their _minAvailable_ field reverted after it's updated by the test, this must be investigated further.

Also noticed that after node reboot it takes some time for respective pods to start so proposed an extra check here - https://github.com/openshift-kni/eco-gotests/pull/413